app/docs/8-CSS.md

```diff
@@ -0,0 +1,56 @@
+# CSS
+
+On our project we are using [SCSS syntax](https://sass-lang.com/guide) and [CSS modules](https://github.com/css-modules/css-modules).
+
+'/src/common/css/common.scss' - common styles.
+
+'/src/common/css/fonts/' - fonts.
+
+'/src/common/css/variables/' - color, font and screen breakpoint variables.
+
+As far as usually components has small size, it's preferable to do not use css selectors nesting (exclude pseudo elements, pseudo classes and state-classes).
+
+It's allowed to use flex-box properties.
+
+Main element of component should have class name the same as component's name, but in lowercase and dash-notation (ComponentName / component-name).
```

Comment:
Speaking of this requirement... Since we use css-modules, I propose to make this as not required, but recommended. Also, we already have complex cases which don't follow this rule, such as`GridRow`.